### ISSUE TITLE
[UX] Allow site admins to specify which menu is to be used in the admin bar.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5519]
This is basically a request to add https://www.drupal.org/project/admin_menu_source into Backdrop core:

![image](https://user-images.githubusercontent.com/2423362/155298816-2cab4def-9571-4500-899d-66db5ed568a5.png)

This would be a nice solution for what this person is trying to achieve in https://forum.backdropcms.org/forum/flyout-menu (aside from the vertical placement of the admin bar).

This is one of the things I'm planning to discuss during the [Content Author-centric UX improvements in core](https://events.backdropcms.org/sessions/content-author-centric-ux-improvements-in-core) session in the upcoming Backdrop Live event.

**Comment by klonos** [comment]
WIP:
![image](https://user-images.githubusercontent.com/2423362/155603811-a57f056c-db8e-4d2e-9934-8c186f6ed213.png)

**Comment by jenlampton** [comment]
I don't think this is an 80% use-case. I think most editors use the admin bar and all the built-in access control checks without any issue. I could see how this would be valuable to those who want something different, so it would be a great contrib project.

**Comment by jenlampton** [comment]
I'm going to brainstorm here (but this will certainly end up being a different issue) but I could see a better solution where a top-level menu item that has only one child link (like "Structure" containing only "Menus") would automatically roll that child item up to the top level. Then, when a new feature is added that allows an editor access to something else, the top-level item would automatically reappear.

**Comment by BWPanda** [comment]
Deja vu: https://github.com/backdrop/backdrop-issues/issues/347
